low-level.lisp

```diff
@@ -105,3 +105,6 @@
           (setf (xfer-pad xfer) 0)
           (setf (ioctl handle spi-ioc-message-1) (cffi:pointer-address xfer))
           read)))))
+
+(defun flush (handle)
+  (force-output handle))
```

Comment:
I don't think this is necessary. The handle is known to be a file descriptor, so stream functions like `force-output` should just be used straight away.